### PR TITLE
* Test against different runtime dependencies versions

### PR DESCRIPTION
**Pull request by PikachuEXE**
+ Add gem `appraisal` as a development dependency

For #25

### CODE REVIEW

File: gemfiles/rails_3_2.gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+      multi_json (~> 1.0)
+      rack (~> 1.0)
+      tilt (~> 1.1, != 1.3.0)
+    sqlite3 (1.3.10)
+    test-unit (3.0.9)
+      power_assert
+    thor (0.19.1)
+    tilt (1.4.1)
+    treetop (1.4.15)
+      polyglot
```
Comment (plentz) [inline gemfiles/rails_3_2.gemfile.lock]: I think that something went wrong

**Comment by PikachuEXE** [comment]
Not sure why that happened, but should be fixed now

**Comment by PikachuEXE** [comment]
Travis config can be changed later for this, want to make sure you accept this change first

File: lol_dba.gemspec [hunk 2]
```diff
@@ -22,4 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activerecord", ">= 3.0", "< 5.0"
   s.add_dependency "actionpack", ">= 3.0", "< 5.0"
   s.add_dependency "railties", ">= 3.0", "< 5.0"
+
+  s.add_development_dependency "appraisal", "~> 1.0"
+  s.add_development_dependency "test-unit"
```
Comment (plentz) [inline lol_dba.gemspec]: why do we need test-unit?

**Comment by plentz** [comment]
btw, isn't simpler to use the solution I proposed in #45? Looks a lot simpler and requires less changes

**Comment by PikachuEXE** [comment]
How do you do it locally? (and easily)
This one only requires running `rake`

**Comment by plentz** [comment]
@PikachuEXE good point :)

**Pull request merged**